.pipelines/run-mirror.yml

```diff
@@ -0,0 +1,30 @@
+# Azure DevOps Pipeline deploying the azure hosted environment
+trigger: none
+pr: none
+
+parameters:
+- name: location
+- name: environment
+  values:
+  - RP-INT
+  - RP-Prod
+- name: DST_AUTH
+- name: SRC_AUTH_GENEVA
+- name: SRC_AUTH_QUAY
+
+jobs:
+  - template: ./templates/template-job-deploy-azure-env.yml
+    parameters:
+      environment: ${{ parameters.environment }}
+      location: ${{ parameters.location }}
+      configFileName: $(config-file-name)
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+      vsoProjectID: $(vso-project-id)
+      vsoConfigPipelineID: $(vso-config-pipeline-id)
+      vsoDeployerPipelineID: $(vso-deployer-build-id)
+  - template: ./templates/template-run-mirror.yml
+    parameters:    
+      DST_AUTH: ${{ parameters.DST_AUTH }}
```

Comment:
these should look like `dstAuth: $(acr-push-auth)` - i.e. `camelCaseName: $(secret-name)`